api/client/cli.go

```diff
@@ -91,19 +91,24 @@ func (cli *DockerCli) Subcmd(name, signature, description string, exitOnError bo
 		errorHandling = flag.ContinueOnError
 	}
 	flags := flag.NewFlagSet(name, errorHandling)
+	options := ""
+	if signature != "" {
+		signature = " " + signature
+	}
+	if flags.FlagCountUndeprecated() > 0 {
```

Comment:
May help to move this up underneath the `options` declaration... or move `options := ""` just above this instead.